core/src/main/java/org/apache/brooklyn/core/mgmt/ha/HighAvailabilityManagerImpl.java

```diff
@@ -456,6 +460,12 @@ else if (startMode==HighAvailabilityMode.HOT_STANDBY && getNodeState()==Manageme
             registerPollTask();
     }
 
+    protected void updatePlaneId(ManagementPlaneSyncRecord existingMaster) {
```

Comment:
Renaming to `updateLocalPlaneId`. It would be surprising to update with a different planeId, but I decided to support the use case where someone goes and edits the persisted state planeId. In case the planeId is different it will log a warning, but will accept the change.